main.py

```diff
@@ -183,6 +183,14 @@ def update(self) -> Image:
             'sha256:abb79827bb0b10100e072c767fe2bb99f25249100020bbe0565fc75e71a78af0',  # noqa: E501
         ),
     ),
+    Image(
+        registry='registry-1.docker.io',
+        source='altinity/clickhouse-server',
+        tag='24.8.11.51285.altinitystable',
+        digests=(
+            'sha256:694d6cce8ed63a4d2677d571f8399a73a53a6e070a7b3555bb551ece522dc649',
```

Comment:
I expect multiple digests here -- if you run the script to update the image it should populate the other architecture as well

Reply:
Yes, I realized I put the index digest here, not the different manifest digests.